src/external/lib/connectors/crm.js

```diff
@@ -162,7 +157,7 @@ async function verify (entityId, verificationCode) {
   }
 
   // Verify with code
```

Comment:
I have added a Trello card for moving the registration logic to the water service